content/applications/general/users/2fa.rst

```diff
@@ -1,127 +1,123 @@
 =========================
-Two-factor Authentication
+Two-factor authentication
 =========================
 
-Two-factor authentication ("2FA") is a good way to improve the
-security of an account, to make it less likely that an other person
-will manage to log in instead of you.
+.. |2fa| replace:: :abbr:`2FA (two-factor authentication)`
+.. |QR| replace:: :abbr:`QR (Quick Response)` code
 
-Practically, it means storing a secret inside an *authenticator*
-(usually your cell phone) and exchanging a code from the authenticator
-when you try to log in.
+Two-factor authentication (2FA) is a good way to improve security, and to prevent unauthorized
+persons from accessing user accounts.
```

Comment:
```suggestion
*Two-factor authentication (2FA)* is a way to improve security, and prevent unauthorized
persons from accessing user accounts.
```
suggesting the use of italics here because this sentence reads like a definition...which we typically format with italics surrounding the main term.
Also, opted to remove "good" to avoid any editorializing